components/activity_and_insights/insights/top_dms_and_new_members/top_dms_table/top_dms_table.tsx

```diff
@@ -173,18 +179,35 @@ const TopDMsTable = (props: Props) => {
     }, [topDMs]);
 
     return (
-        <DataGrid
-            columns={getColumns}
-            rows={getRows}
-            loading={loading}
-            page={0}
-            nextPage={() => {}}
-            previousPage={() => {}}
-            startCount={1}
-            endCount={10}
-            total={0}
-            className={classNames('InsightsTable', 'TopDMsTable')}
-        />
+        <>
+            <DataGrid
+                columns={getColumns}
+                rows={getRows}
+                loading={loading}
+                page={0}
+                nextPage={() => {}}
+                previousPage={() => {}}
+                startCount={1}
+                endCount={10}
+                total={0}
```

Comment:
Are these all meant to be hard-coded or no-ops?

Reply:
These are meant to be hardcoded because our pagination is quite different